meta.go

```diff
@@ -1,20 +1,192 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
+	"io"
+	"io/ioutil"
 	"os"
+	"reflect"
 	"runtime/debug"
+	"strconv"
 
 	"github.com/urfave/cli"
 )
 
 // VERSION gets set by the build script via the LDFLAGS
 var VERSION string
 
-func getMeta(key string, metaSpace string) {
+var mkdirAll = os.MkdirAll
+var stat = os.Stat
+var writeFile = ioutil.WriteFile
+var readFile = ioutil.ReadFile
+var fprintf = fmt.Fprintf
+
+const metaFile = "meta.json"
+
+// Get meta from file based on key
+func getMeta(key string, metaSpace string, output io.Writer) {
+	metaFilePath := metaSpace + "/" + metaFile
+	metaJson, err := readFile(metaFilePath)
+	if err != nil {
+		panic(err)
```

Comment:
I see. In the case of `meta-cli`, I think it should have non-zero exit code.
Fixed about exit code.